runtime/src/accounts_db.rs

```diff
@@ -6189,7 +6189,7 @@ pub mod tests {
             if let Some((mut account, _)) =
                 accounts.load_without_fixed_root(&ancestors, &pubkeys[idx])
             {
-                account.lamports += 1;
+                account.checked_add_lamports(1).unwrap();
```

Comment:
if this overflows we have bigger problems but anytime we add an upwrap to core code flow it makes me nervous

Reply:
This is test code, right? What do you suggest that I do? Ultimately some test will panic, eat the problem, fail, ...

Reply:
> if this overflows we have bigger problems but anytime we add an upwrap to core code flow it makes me nervous

This is a helper in `mod tests`, it almost go me too :slightly_smiling_face: